services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -1013,7 +1013,8 @@
                 "entityUnfinishedMessage": "Complete the remaining indicators for this access measure by entering details.",
                 "enterEntityDetailsButtonText": "Enter details",
                 "editEntityDetailsButtonText": "Edit details",
-                "drawerTitle": "Add details for access measure"
+                "drawerTitle": "Add details for access measure",
+                "drawerReminder": "Complete all fields and select the Save & close button to save this section."
```

Comment:
Blocking: Changes made in `mcpar.json` are forward-looking only, so this will not apply to any report that has already been created. Applying it retroactively would require an ETL.

So we should not be putting this text here, and instead should be putting it in the drawer verbiage file. Note: We expect it to the be the same for every drawer.

Reply:
@BearHanded @ailZhou

Reply:
fixed, text is now being called from constants.ts